test_regress/t/t_vpi_module.cpp

```diff
@@ -192,6 +192,7 @@ int main(int argc, char** argv) {
     const std::unique_ptr<VM_PREFIX> topp{new VM_PREFIX{contextp.get(),
                                                         // Note null name - we're flattening it out
                                                         ""}};
+    contextp->scopesDump();
```

Comment:
Remove, it's already below, you can use --verbose when running a test to see it.